service/metrics/metrics.go

```diff
@@ -231,13 +238,13 @@ func (m *shadowsocksMetrics) AddTCPProbe(clientLocation, status, drainResult str
 
 func (m *shadowsocksMetrics) AddUDPPacketFromClient(clientLocation, accessKey, status string, clientProxyBytes, proxyTargetBytes int, timeToCipher time.Duration) {
```

Comment:
The `status` argument appears to be unused now, because (unlike TCP) there is no longer any metric monitoring UDP status codes.  This seems slightly worrying, both for probes and for potential bugs.

If you don't think we need to monitor UDP status codes, then we should probably remove this argument.

Reply:
I've restored the status. I need to think more about it, but I want to get the rest of the changes in for the next release.